tiatoolbox/tools/pyramid.py

```diff
@@ -430,3 +433,251 @@ def tile_path(self, level: int, x: int, y: int) -> Path:
+        return int(total_level_count)
+
+    def get_tile(
+        self,
+        level: int,
+        x: int,
+        y: int,
+        pad_mode: str = None,
+        interpolation: str = None,
+    ) -> Image:
+        """Render a tile at a given level and coordinate.
+
+        Note that levels are in the reverse order of those in WSIReader.
+        I.E. level 0 here corresponds to the lowest resolution whereas
+        level 0 in WSIReader corresponds to the maximum resolution
+        (baseline).
+
+        Args:
+            level (int):
+                The pyramid level of the tile starting from 0 (the whole
+                slide in one tile, 0-0-0).
+            x (int):
+                The tile index in the x direction.
+            y (int):
+                The tile index in the y direction.
+
+        Returns:
+            Image:
+                Pillow image of the tile.
+
+        Example:
+            >>> from tiatoolbox.tools.pyramid import AnnotationTileGenerator
+            >>> from tiatoolbox.wsicore.wsireader import WSIReader
+            >>> from tiatoolbox.annotation.storage import SQLiteStore
+            >>> wsi = WSIReader.open("sample.svs")
+            >>> SQ=SQLiteStore.from_geojson(geo_path)
+            >>> tile_generator = AnnotationTileGenerator(
+            ...   info=wsi.info,
+            ...   store=SQ,
+            ... )
+            >>> tile_0_0_0 = tile_generator.get_tile(level=0, x=0, y=0)
+
+        """
+        if pad_mode is not None or interpolation is not None:
+            warnings.warn(
+                "interpolation, pad_mode are unused by AnnotationTileGenerator"
+            )
+        if level < 0:
+            raise IndexError
+        if level > self.level_count:
+            raise IndexError("Invalid level.")
+
+        scale = self.level_downsample(level)
+        baseline_x = (x * self.tile_size * scale) - (self.overlap * scale)
+        baseline_y = (y * self.tile_size * scale) - (self.overlap * scale)
+        output_size = [self.output_tile_size] * 2
+        coord = [baseline_x, baseline_y]
+        if level < self.sub_tile_level_count:
+            output_size = self.output_tile_size // 2 ** (
+                self.sub_tile_level_count - level
+            )
+            output_size = np.repeat(output_size, 2).astype(int)
+            thumb = self.get_thumb_tile()
+            thumb.thumbnail(output_size)
+            return thumb
+        slide_dimensions = np.array(self.info.slide_dimensions)
+        if all(slide_dimensions < [baseline_x, baseline_y]):
+            raise IndexError
+
+        rgb = np.zeros((*output_size, 4), dtype=np.uint8)
+        bounds = locsize2bounds(coord, [self.tile_size * scale] * 2)
+        bound_geom = Polygon.from_bounds(*bounds)
+        rgb = self.render_annotations(rgb, bound_geom, scale, coord)
+
+        return Image.fromarray(rgb)
+
+    def render_by_type(
+        self,
+        rgb: np.ndarray,
+        ann: Annotation,
```

Comment:
Why not just call this `annotation` to be more clear?

Reply:
sure, it can be changed to annotation